examples/Notebooks/dev/flopy3_dev_mf6_packaging_distributing_flopy_plugins.ipynb

```diff
@@ -0,0 +1,234 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "b4c70565",
+   "metadata": {},
+   "source": [
+    "# Distributing your FloPy Plugin as a Python Package\n",
+    "\n",
+    "FloPy plug-ins can be installed as python packages.  FloPy will automatically detect plug-ins in other python packages by using python package metadata.\n",
+    "\n",
+    "In this notebook you will package an existing FloPy plug-in in its own python package and then run an example simulation using that plug-in.  The plug-in being packaged is generated in the flopy3_dev_mf6_BuildStressPlugin notebook. Please run that notebook first.\n",
+    "\n",
+    "First, you need to determine the plug-in extension your FloPy plug-in is using.  This is necessary to identify the files you wish to export.  Also, choose a name for the python package you wish to create (if you do not specify a name, a name will be created for you)."
```

Comment:
This is confusing.  Is this extension or abbreviation?  Not sure what to do about this.  I see you've adopted extension fairly widely.  And maybe this is okay, but it might be good to say the extension is an abbreviation for the plugin that users will see.  It is similar to RIV for River or SFR for Stream Flow Routing in MODFLOW terminology, for example.